scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-design-components.adoc

```diff
@@ -35,28 +34,30 @@ include::modules/telco-core-rds-networking.adoc[leveloffset=+1]
 
 * xref:../../../networking/understanding-networking.adoc#understanding-networking[Understanding networking]
 
-include::modules/telco-core-cluster-network-operator.adoc[leveloffset=+2]
+include::modules/telco-core-cluster-network-operator.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * xref:../../../networking/cluster-network-operator.adoc#nw-cluster-network-operator_cluster-network-operator[Cluster Network Operator]
 
-include::modules/telco-core-load-balancer.adoc[leveloffset=+2]
+include::modules/telco-core-load-balancer.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * xref:../../../networking/metallb/about-metallb.adoc#about-metallb[About MetalLB and the MetalLB Operator]
 
-include::modules/telco-core-sriov.adoc[leveloffset=+2]
+include::modules/telco-core-sriov.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * xref:../../../networking/hardware_networks/about-sriov.adoc#about-sriov[About Single Root I/O Virtualization (SR-IOV) hardware networks]
 
-include::modules/telco-nmstate-operator.adoc[leveloffset=+2]
+* link:../../../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices]
```

Comment:
Page not found with this xref.

Reply:
This is a link, should be an xref. Updated.